### PR TITLE
py-ansible: update to 2.9.17

### PR DESCRIPTION
**Pull request by tsabirgaliev**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.1 20D74
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-ansible/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-ansible
-version             2.9.10
+version             2.9.17
```
Comment (reneeotten) [inline python/py-ansible/Portfile:8]: why not updating to the latest version (2.10.7)?

Comment (tsabirgaliev) [reply python/py-ansible/Portfile:8]: 2.10.x has some major changes, and just updating version didn't work on my machine. Unfortunately I'm not python expert enough to resolve the issues.

Comment (adfernandes) [reply python/py-ansible/Portfile:8]: looking into it right now

Comment (adfernandes) [reply python/py-ansible/Portfile:8]: Will be closed by https://github.com/macports/macports-ports/pull/10063

**Pull request closed without merging**